php/php.editor/src/org/netbeans/modules/php/editor/csl/DeclarationFinderImpl.java

```diff
@@ -397,6 +402,29 @@ private OffsetRange getVarCommentOffsetRange(TokenSequence<PHPTokenId> ts, Strin
             return null;
         }
 
+        private Set<OffsetRange> getTypeRanges(String types, int startOffset) {
+            String[] splitTypes = Type.splitTypes(types);
+            Set<OffsetRange> ranges = new HashSet<>(splitTypes.length);
```

Comment:
Nitpick: please note that for `Set` (and `Map`), this ctor is a bit different in comparison to the `List` one (sets or maps have load factor, by default 0.75, so actually, the initial size should be bigger).

Reply:
I see. maybe types are 12 or less in most cases.
So, I'll use the constructor without parameters. (16*0.75 = 12)
Thanks!

Reply:
Or simply use `size * 1.5` and you should be safe, always :)

Reply:
I'll change the size to it :) Thank you, Tomas!

Reply:
My pleasure, thanks!